litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml

```diff
@@ -203,7 +203,7 @@ rules:
 
 - apiGroups: ["apps"]
   resources: ["deployments", "daemonsets", "replicasets", "statefulsets"]
-  verbs: ["get","list", "delete"]
+  verbs: ["get","list", "delete","deletecollection"]
```

Comment:
Please add same permission here for cluster here - https://github.com/litmuschaos/litmus/blob/ce5b7dd806eed34451f85607daed43ef5c14331a/litmus-portal/manifests/cluster-k8s-manifest.yml#L307